Makefile

```
@@ -189,7 +207,12 @@ build/config.json:
 # ESP32 VARIANTS FLASH
 .PHONY: flash
 flash: check-env-flash sdk esp32
+ifdef ESP32_EXTRA_PROGRAM
+flash: build/$(ESP32_CHIP)/programs.bin
+	python $(IDF_PATH)/components/esptool_py/esptool/esptool.py --chip $(ESP32_CHIP) --port $(ESP32_PORT) --baud 921600 --before default_reset --after hard_reset write_flash -z --flash_mode dio --flash_freq 40m --flash_size detect 0x1000 build/$(ESP32_CHIP)/bootloader/bootloader.bin 0x10000 build/$(ESP32_CHIP)/toit.bin 0x8000 build/$(ESP32_CHIP)/partitions.bin 0x200000 build/$(ESP32_CHIP)/programs.bin
+else
 	python $(IDF_PATH)/components/esptool_py/esptool/esptool.py --chip $(ESP32_CHIP) --port $(ESP32_PORT) --baud 921600 --before default_reset --after hard_reset write_flash -z --flash_mode dio --flash_freq 40m --flash_size detect 0x1000 build/$(ESP32_CHIP)/bootloader/bootloader.bin 0x10000 build/$(ESP32_CHIP)/toit.bin 0x8000 build/$(ESP32_CHIP)/partitions.bin
+endif
```

Alternatively the following might be nicer.
(Untested):
```suggestion
PROGRAM_PARTITION_FLAGS=
ifdef ESP32_EXTRA_PROGRAM
PROGRAM_PARTITION_FLAGS=0x200000 build/$(ESP32_CHIP)/programs.bin
endif

.PHONY: flash
flash: check-env-flash sdk esp32
flash: build/$(ESP32_CHIP)/programs.bin
	python $(IDF_PATH)/components/esptool_py/esptool/esptool.py --chip $(ESP32_CHIP) --port $(ESP32_PORT) --baud 921600 --before default_reset --after hard_reset write_flash -z --flash_mode dio --flash_freq 40m --flash_size detect 0x1000 build/$(ESP32_CHIP)/bootloader/bootloader.bin 0x10000 build/$(ESP32_CHIP)/toit.bin 0x8000 build/$(ESP32_CHIP)/partitions.bin $(PROGRAM_PARTITION_FLAGS)
```